<?xml version="1.0"?>
<launch>
  <arg name="debug" default="false" />
  <arg unless="$(arg debug)" name="launch_prefix" value="" />
  <arg if="$(arg debug)" name="launch_prefix" value="gdb --ex run --args" />
  <rosparam file="$(find odrive_control)/config/odrive_controllers.yaml" command="load" />

  <!-- Load example URDF -->
  <param name="robot_description" command="$(find xacro)/xacro '$(find robo_description)/urdf/URDF'" />
  <!-- This is required for publishing the dynamic tf transforms -->
  <!--  Load the robot_state_publisher node  -->
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" /> 

  <!-- Load the odrive_control node -->
  <!-- <node name="odrive_control" pkg="odrive_control" type="odrive_control"  output="screen" launch-prefix="$(arg launch_prefix)"  /> -->

  <!-- Load the yanthra_test140 node -->
  <rosparam file="$(find yanthra_move)/config/yanthra_move_test140.yaml" command="load" />
  <arg name="output_log" default="log" />
  <node name="yanthra_test140_node" pkg="yanthra_move" type="yanthra_test140"
    output="$(arg output_log)"  />

</launch>
